sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -1443,4 +1443,30 @@ public String getSnapshotId() {
     public boolean isSnapshot() {
         return this.snapshot != null;
     }
+
+    /**
+     * Get the container name.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.BlobAsyncClient.getContainerName}
+     *
+     * @return The name of the container.
+     */
+    public String getContainerName() {
```

Comment:
Should we think about marking these methods as final to prevent them being overridden in a subclass?

Reply:
Agreed. added